index.bs

```diff
@@ -33,29 +33,100 @@ Introduction {#intro}
-
-Thus, it is not possible for a content author to control <a>scroll chaining</a> and overscroll in a robust, performant and forward compatible way. The 'scroll-boundary-behavior' property fixes this shortcoming.
-
-<pre class=example>
-A 'position: fixed' left navigation bar may not want to hand off scrolling to the
-document but does not want to prevent native overscroll affordances.
-</pre>
+A content author does not necessarily want <a>scroll chaining</a> to occur for all <a>scroll
+containers</a>. Some <a>scroll containers</a> may be part of a <a>containing block chain</a> but may
+serve a different logical purpose in the document and may want to prevent scrolling from continuing
+up the <a>scroll chain</a>. To achieve this, a content author will install event listeners without
+the <a>passive flag</a> set and will use <a>preventDefault</a> when there is a risk that scroll
+chaining will occur. This is detrimental for the following reasons:
+
+* The user agent may in the future introduce new input methods for scrolling that are not supported
+    by the content author's event listeners.
+* A non passive event listener will delay scrolling because the user agent will have to wait for the
+    result of the event listener to determine if <a>preventDefault</a> was called causing increased
+    scroll latency.
+* When scrolling is performed near the edge of the <a>scroll boundary</a>, the <a>default action</a>
+    may cause both scrolling to the edge of the <a>scroll container</a> and a <a>boundary default
+    action</a>. Calling <a>preventDefault</a> will not only cancel the <a>boundary default action</a>
+    but also the scroll to the edge of the <a>scrollport</a>.
+* The <a>default action</a> for the event may also provide additional behavior that the author does
+    not want to cancel such as an overscroll affordance. <a>preventDefault</a> doesn't allow the
+    content author to cancel only some of the <a>default actions</a> such as scroll chaining.
+
+Thus, it is not possible for a content author to control <a>scroll chaining</a> and overscroll in a
+robust, performant and forward compatible way. The 'scroll-boundary-behavior' property fixes this
+shortcoming.
+
+
+Motivating Examples {#motivating-examples}
+=================
+
+
+<div class=example>
+A 'position: fixed' left navigation bar does not want to hand off scrolling to the document but does
+not want to prevent native overscroll affordances. Note that A scroll gesture performed on the
+navigation bar s is almost never meant to scroll the document.
+
+  <pre class="lang-css">
+  #sidebar {
+    scroll-boundary-behavior: contain;
+  }
+  </pre>
+
+In this case, the author can use <a value for=scroll-boundary-behavior>contain</a> on the sidebar to
+prevent its scroll to chain up to document.
```

Comment:
"to prevent its scroll to chain up to document"
I think the wording can be improved a bit. How about:
'to prevent scrolling from being chained to the parent document element."

Reply:
Done.